apps/server/src/modules/deletion/repo/mapper/deletion-request.mapper.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { DeletionRequest } from '../../domain/deletion-request.do';
+import { deletionRequestFactory } from '../../domain/testing/factory/deletion-request.factory';
+import { DeletionRequestEntity } from '../../entity';
+import { deletionRequestEntityFactory } from '../../entity/testing/factory/deletion-request.entity.factory';
+import { DeletionRequestMapper } from './deletion-request.mapper';
+
+describe(DeletionRequestMapper.name, () => {
+	describe('mapToDO', () => {
+		describe('When entity is mapped for domainObject', () => {
+			it('should properly map the entity to the domain object', () => {
+				const entity = deletionRequestEntityFactory.build();
+
+				const domainObject = DeletionRequestMapper.mapToDO(entity);
+
+				const expectedDomainObject = new DeletionRequest({
+					id: entity.id,
+					domain: entity.domain,
+					deleteAfter: entity.deleteAfter,
+					itemId: entity.itemId,
+					status: entity.status,
+					createdAt: entity.createdAt,
+					updatedAt: entity.updatedAt,
+				});
```

Comment:
Please move it (as well as the entity preparation in the first place) to the `setup()` function.

Reply:
done